felix/bpf-gpl/tc.c

```diff
@@ -725,6 +726,7 @@ static CALI_BPF_INLINE struct fwd calico_tc_skb_accepted(struct cali_tc_ctx *ctx
 		}
 		if (CALI_F_FROM_WEP && state->flags & CALI_ST_SKIP_FIB) {
 			ct_ctx_nat.flags |= CALI_CT_FLAG_SKIP_FIB;
+			seen_mark = CALI_SKB_MARK_SKIP_FIB;
```

Comment:
this must have been already set @573, right? Here we only mark the connection so that we always make the flow to skip fib. Perhaps it should be also done where the mark is set?